src/_playground/documentation/Properties/defaults.js

```diff
@@ -8,6 +8,7 @@ export const defaultPropDescriptions = {
     id: 'Value for the `id` attribute on the element.',
     inputProps: 'Additional props to be spread to the `<input>` element.',
     labelProps: 'Additional props to be spread to the `<label>` element.',
+    level: 'Heading level. `<h1>` is not allowed for accessibility reasons.',
```

Comment:
this might be better...

`<H1>` is reserved for the page title. It should not appear in components.

..that said, seems like the actionbar *could* be used as a page title. Thoughts?

Reply:
I wondered the same thing about `ActionBar`.  Not quite sure what the design team was envisioning for that component, but it does seem like it _could_ be used as a page title.